### ISSUE TITLE
Problem with npm run restore 

### ISSUE THREAD

**Issue by ryan-0** [shprink/wordpress-hybrid-client#241]
Hi, 
I'm having this error:
<img width="645" alt="screen shot 2016-06-04 at 5 05 12 pm" src="https://cloud.githubusercontent.com/assets/11512435/15802110/90794244-2a76-11e6-83c6-5f9832ce61c7.png">
I'm running npm 3.9.5 and I used `npm run restore ` 
Any idea how to fix this?
Thank you!

**Comment by maxlinux2000** [comment]
try to change the cordova android ad a problematic plugin

```
cordova platform  remove android
cordova platform  add android
cordova plugin remove nl.x-services.plugins.socialsharing
cordova plugin add cordova-plugin-x-socialsharing
cordova prepare
```

**Comment by shprink** [comment]
Fixed with 2.0.0-beta09
